src/gui/dialogs/achievements_dialog.cpp

```diff
@@ -122,4 +122,4 @@ void achievements_dialog::set_achievements_content()
 	}
 }
 
-} // namespace gui2::dialogs
```

Comment:
how is this related to the Elvish Hunter-line PR?

Reply:
Looks like it got picked up from the fix-whitespace script.